### DISCUSSION TITLE
Catalyst 21.10

### DISCUSSION THREAD

**Discussion by Scitator** [catalyst-team/catalyst#1343] [Announcements]
### Tl;dr

Readmes and tutorials with a few ddp fixes.

### Added

- RSquareLoss ([#1313](https://github.com/catalyst-team/catalyst/pull/1313))
- Self-Supervised example updates: ([#1305](https://github.com/catalyst-team/catalyst/pull/1305)), ([#1322](https://github.com/catalyst-team/catalyst/pull/1322)), ([#1325](https://github.com/catalyst-team/catalyst/pull/1325)), ([#1335](https://github.com/catalyst-team/catalyst/pull/1335))
- Albert training example ([#1326](https://github.com/catalyst-team/catalyst/pull/1326))
- YOLO-X (new) detection example and refactoring ([#1324](https://github.com/catalyst-team/catalyst/pull/1324))
- `TopKMetric` asbtraction ([#1330](https://github.com/catalyst-team/catalyst/pull/1330))

### Changed

- simlified readme ([#1312](https://github.com/catalyst-team/catalyst/pull/1312))
- improved DDP tutorial ([#1327](https://github.com/catalyst-team/catalyst/pull/1327))
- `CMCMetric` renamed from `<prefix>cmc<suffix><k>` to `<prefix>cmc<k><suffix>` ([#1330](https://github.com/catalyst-team/catalyst/pull/1330))

### Removed

-

### Fixed

- Zero seed error ([#1329](https://github.com/catalyst-team/catalyst/pull/1329))
- updated codestyle issues ([#1331](https://github.com/catalyst-team/catalyst/pull/1331))
- TopK metrics: ([#1330](https://github.com/catalyst-team/catalyst/pull/1330)), ([#1334](https://github.com/catalyst-team/catalyst/pull/1334)), ([#1339](https://github.com/catalyst-team/catalyst/pull/1339))
- `--expdir` param for `catalyst-dl run` ([#1338](https://github.com/catalyst-team/catalyst/pull/1338))
- ControlFlowCallback for distributed setup ([#1341](https://github.com/catalyst-team/catalyst/pull/1341))

<hr /><em>This discussion was created from the release <a href='https://github.com/catalyst-team/catalyst/releases/tag/v21.10'>Catalyst 21.10</a>.</em>
